#71#72#73#74 - Contrast Minimum - Based on Master

bae-junghwan opened the pull request:
Hello! This PR addresses Issues #71 , #72 , #73 , and #74 . In summary, links on pages have had their font colors changed to be a minimum of 4.5:1 contrast ratio. In addition, the navigation bar now has a green background, gray border, and white text to make the contrast ratio compliant as well.

### Changes made:
**index.html**
- Navigation bar now has a green background with a gray border, similar to the one on DATIM
- Items in navbar changed to white font color
- The email link is now a contrast ratio compliant blue

**LoadedData.html**
- Navigation bar now has a green background with a gray border, similar to the one on DATIM
- Items in navbar changed to white font color
- The link is now a contrast ratio compliant blue

**SpecialRequests.html**
- Navigation bar now has a green background with a gray border, similar to the one on DATIM
- Items in navbar changed to white font color
- The link is now a contrast ratio compliant blue

**WhatsNext.html**
- Navigation bar now has a green background with a gray border, similar to the one on DATIM
- Items in navbar changed to white font color
- The link is now a contrast ratio compliant blue

**style.css**
- Added rules needed for compliant contrast ratio
  - Green background + gray border for navigation bar
  - Compliant blue font color for links
---
**index.html**
![image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/123202460/c6518354-051e-4fa7-a426-e3f6ff1c610b)
---
**LoadedData.html**
![image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/123202460/c1ec4f84-a5b0-438d-a66a-0ec219c5a00b)
---
**SpecialRequests.html**
![image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/123202460/84db7863-8260-4ecb-ba50-d4432cc474b4)
---
**WhatsNext.html**
![image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/123202460/45d77433-c8e5-4e6a-b156-126ca5498a14)

Kelvyn-Meyers commented on LoadedData.html:
```diff
@@ -11,30 +11,33 @@
-    </ul>
-  </div>
-</nav>
+
+<div class="nav-container">
+  <nav class="navbar navbar-expand-lg navbar-light">
+    <a class="navbar-brand" href="" style="color:white;">DATIM</a>
```
A few things here:
1. On the Loaded Data page, the "Plan Revision Statuses for NFS" link still has color contrast errors and needs that new class you created applied to it.
2. I see all of these navigation links in the header have `color: white;` applied to them. All of these links also have a class applied name `nav-link`. Could you just add a CSS rule for `nav-link` that applies this color instead of doing it manually per link? This way, if we ever need to make changes to the the navigation link color again, it'll hopefully be a one-line change.
3. In the ETL dialogs, the X button needs to be manually tested. Assuming I tested correctly, it seems like this X button fails when not hovered. Could you please fix this one, too?
4. If you zoom in far enough on these pages, a hamburger menu appears that lets you toggle the various links. Could you apply a similar coloring to these, too? This one seems to apply color a bit weirdly, so let me know if you need help figuring this part out.
![image](https://github.com/UNLV-FIA/DATIM-ETL-Priorities-and-Communication-Plan/assets/72825162/bbb50d4a-9a4e-4398-815d-a49c039f3407)

bae-junghwan commented:
Hi @Kelvyn-Meyers , I've made the changes requested.

- Added the `text-link` class to the link in Loaded Data
- Removed `style="color:white;"` and added it as a rule to `nav-link` items in styles.css
- The X button in ETL should be compliant now
- The hamburger menu has been switched to the lighter version

Kelvyn-Meyers reviewed (approved):
Excellent work here @bae-junghwan! These changes have resolved all the remaining color contrast issues and I think made the site look great! Ready to merge.
